rust/catalyst-voting/benches/vote_protocol.rs

```diff
@@ -0,0 +1,195 @@
+//! `catalyst_voting::vote_protocol` benchmark
+#![allow(
+    missing_docs,
+    clippy::missing_docs_in_private_items,
+    clippy::unwrap_used,
+    clippy::similar_names
+)]
+
+use catalyst_voting::{
+    crypto::default_rng,
+    vote_protocol::{
+        committee::{ElectionPublicKey, ElectionSecretKey},
+        tally::{
+            decrypt_tally,
+            proof::{generate_tally_proof, verify_tally_proof},
+            tally, DecryptionTallySetup,
+        },
+        voter::{
+            encrypt_vote,
+            proof::{generate_voter_proof, verify_voter_proof, VoterProofCommitment},
+            Vote,
+        },
+    },
+};
+use criterion::{criterion_group, criterion_main, Criterion};
+use proptest::{
+    prelude::{any, Strategy},
+    strategy::ValueTree,
+    test_runner::TestRunner,
+};
+use test_strategy::Arbitrary;
+
+const VOTING_OPTIONS: usize = 3;
+const VOTERS_NUMBER: usize = 100;
+
+#[derive(Arbitrary, Debug)]
+struct Voter {
+    voting_power: u32,
+    #[strategy(0..VOTING_OPTIONS)]
+    choice: usize,
+}
+
+fn initial_setup() -> (
```

Comment:
It's hard to understand what data `Vec<usize>, Vec<u64>,` hold. 
We could add comments to the function. What do you think?

Reply:
I would rather add type aliases with the proper naming, but in general yea, need to be more clear at this place

Reply:
If you want to go the extra mile, I'd propose creating a concrete type (e.g., a new type pattern) instead of type aliases.